### PR TITLE
Add follower and following view

### PR DESCRIPTION
**Pull request by BichVN**

### CODE REVIEW

File: app/controllers/followers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class FollowersController < ApplicationController
+  def index
+    @title = t("followers")
```
Comment (yuyuvn) [inline app/controllers/followers_controller.rb]: remove ()

File: app/controllers/followings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class FollowingsController < ApplicationController
+  def index
+    @title = t("following")
```
Comment (yuyuvn) [inline app/controllers/followings_controller.rb]: not need

File: db/migrate/20150730082940_create_relationships.rb [hunk 3]
```diff
@@ -1,10 +0,0 @@
-class CreateRelationships < ActiveRecord::Migration
```
Comment (yuyuvn) [inline db/migrate/20150730082940_create_relationships.rb]: don't change migrate file

File: app/views/shared/_show_follow.html.erb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      <%= render "shared/stats" %>
+    </section>
+  </aside>
+  <div class="col-md-8">
+    <h3><%= @title %></h3>
+    <% if @users.any? %>
+      <ul class="users follow">
+        <%= render @users %>
+      </ul>
+    	<%= will_paginate @users %>
```
Comment (yuyuvn) [inline app/views/shared/_show_follow.html.erb]: indent

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,6 +1,13 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
   include SessionsHelper
+
+  private
+
```
Comment (ThuBM) [inline app/controllers/application_controller.rb]: remove line

**Comment by ThuBM** [comment]
OK

**Pull request merged**